bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/VBS_WWjj_SS/WWjj_SS_ewk_qcd_proc_card.dat

```diff
@@ -0,0 +1,38 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.2.1                 2014-09-25         *
```

Comment:
please remove the banner and in general follow these recommendations
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

Reply:
Done